kevinhartmann23/fitlit
# Feature/dom manipulation

#### What does  this PR do?

This PR implements DOM manipulation, and the functionality needed to gather and interpolate necessary data into its respective elements.

#### Where should the reviewer start?

Familiarize yourself with the domHelpers.js file, and how it interacts with the newly created data helper functions. Additionally, look at the date input element added to index.html.

#### Is this a bug fix or a feature?

Feature.

#### How should this be manually tested?

In browser, login as a user, and check that the hydration, activity, and sleep pages are displaying properly.
